### PR TITLE
Added unit tests for ThemeService

### LINKED ISSUE #384: Add tests for ThemeService
Currently, there are no unit tests for `ThemeService`. We need to implement unit tests for this service and cover it's public methods.

### PR DESCRIPTION
**Pull request by Vso1o**

### CODE REVIEW

**Comment by andrii-developer** [comment]
Please follow best practices for [Naming your tests](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests) and [Avoid multiple asserts](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#avoid-multiple-asserts).

File: CharlieBackend.Api.UnitTes/ThemeServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+                Name = "Test_name"
+            };
+
+            var expectedThemeDto = new ThemeDto()
+            {
+                Id = 10,
+                Name = "Test_name"
+            };
+
+            var themeRepositoryMock = new Mock<IThemeRepository>();
```
Comment (gingerbread-monster) [inline CharlieBackend.Api.UnitTes/ThemeServiceTests.cs]: Please, always avoid the repetitive code.
Move this line from each test method into class constructor.

File: CharlieBackend.Api.UnitTes/ThemeServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+            themeRepositoryMock.Setup(x => x.GetThemeByNameAsync("Test_name"))
+                .ReturnsAsync(existingTheme);
+
+            _unitOfWorkMock.Setup(x => x.ThemeRepository).Returns(themeRepositoryMock.Object);
+
+            var themeService = new ThemeService(
+                _unitOfWorkMock.Object,
+                _mapper);
```
Comment (gingerbread-monster) [inline CharlieBackend.Api.UnitTes/ThemeServiceTests.cs]: This can be moved from each test method into class constructor as well.

**Review by DemetriosKoras — APPROVED** [review]

File: CharlieBackend.Api.UnitTes/ThemeServiceTests.cs [hunk 3]
```diff
@@ -20,12 +20,19 @@ public class ThemeServiceTests : TestBase
...
         public ThemeServiceTests()
         {
-            _notificationServiceMock = new Mock<INotificationService>();
             _mapper = GetMapper(new ModelMappingProfile());
+            _themeRepositoryMock = new Mock<IThemeRepository>();
+        }
+
+        private void Initialize()
+        {
+            _themeService = new ThemeService(
```
Comment (Pasha1968) [inline CharlieBackend.Api.UnitTes/ThemeServiceTests.cs:30]: You probably  can move it to constructor and not to invoke 'Initialize' every time

**Review by Pasha1968 — APPROVED** [review]

**Review by andrii-developer — APPROVED** [review]

**Pull request merged**